src/components/drawer/drawer.html

```diff
@@ -1,58 +1,45 @@
 <aside class="drawer mdc-drawer--open mdc-drawer--persistent mdc-typography">
-  <nav class="mdc-drawer__drawer">
+  <nav class="mdc-drawer__drawer drawer__nav">
     <div class="drawer__group mdc-list-group">
-      <nav class="drawer__list mdc-list">
+      <nav class="drawer__list  mdc-list--non-interactive">
         <a class="drawer__list-item mdc-list-item drawer__add-account-dialog-activation" href="#">
           <i class="drawer__icon material-icons mdc-list-item__graphic" aria-hidden="true">add</i>
           Add new account
         </a>
         <a class="drawer__list-item mdc-list-item" href="#">
-          <i class="drawer__icon material-icons mdc-list-item__graphic" aria-hidden="true">account_balance</i>Privat
-          -
-          $200
-          <button class="drawer__more mdc-button">
-            <i class="drawer__more-icon  material-icons"
-              title="More options">more_vert</i></button>
+          <i class="drawer__icon material-icons mdc-list-item__graphic" aria-hidden="true">account_balance</i>Privat - $200
+
+          <div class="drawer__mount-point"></div>
```

Comment:
why this class name? in table transactions it is "table-transactions__more-button", isn't it?